src/docs/testing/ui-performance/index.md

```diff
@@ -470,28 +448,55 @@ integration test framework, you can generate metrics to track the following:
-* [Flutter API]({{site.api}}/index.html) docs, particularly the
-  [PerformanceOverlay]({{site.api}}/flutter/widgets/PerformanceOverlay-class.html)
-  class, and the
-  [dart:developer]({{site.api}}/flutter/dart-developer/dart-developer-library.html)
-  package.
-
-[IntelliJ Flutter Inspector Window]: {% asset tools/android-studio/visual-debugging.png @path %}
+* [Debugging Flutter apps][]
+* [Widget inspector][]
+* [Flutter Inspector talk][], presented at DartConf 2018
+* [Why Flutter Uses Dart][], an article on Hackernoon
+* [Observatory][]: A Profiler for Dart Apps
+* [Flutter API][] docs, particularly the [PerformanceOverlay][] class, and the
+  [dart:developer][] package
+
+[IntelliJ Flutter inspector window]: {% asset tools/android-studio/visual-debugging.png @path %}
+
+[RepaintBoundary]: {{site.api}}/flutter/widgets/RepaintBoundary-class.html
+[Tracing any Dart code performance]: /docs/testing/debugging#tracing-any-dart-code-performance
+[Debugging Flutter apps]: /docs/testing/debugging
+[examples]: {{site.github}}/flutter/flutter/tree/master/examples/flutter_gallery
+[Observatory]: https://dart-lang.github.io/observatory/
+[Flutter's build modes]: /docs/testing/build-modes
+[profile mode]: /docs/testing/build-modes#profile
+[PerformanceOverlay]: {{site.api}}/flutter/widgets/PerformanceOverlay-class.html
+[dart:developer]: {{site.api}}/flutter/dart-developer/dart-developer-library.html
+[PerformanceOverlayLayer.checkerboardOffscreenLayers]: {{site.api}}/flutter/rendering/PerformanceOverlayLayer/checkerboardOffscreenLayers.html
+[PerformanceOverlayLayer.checkerboardRasterCacheImages]: {{site.api}}/flutter/rendering/PerformanceOverlayLayer/checkerboardRasterCacheImages.html
+[Flutter API]: {{site.api}}
+[UIKit]: https://developer.apple.com/documentation/uikit
+[MainThread]: {{site.android-dev}}/reference/android/support/annotation/MainThread
+[Integration testing]: /docs/testing#integration-testing
+[Architecture notes]: {{site.github}}/flutter/engine/wiki#architecture-notes
+[Widget inspector]: /docs/development/tools/inspector
+[Flutter Inspector talk]: https://www.youtube.com/watch?v=JIcmJNT9DNI
+[bookshelf-like icon]: /docs/testing/ui-performance/images/performance-overlay-icon.png
+[line-chart icon]: /docs/testing/ui-performance/images/observatory-timeline-icon.png
+[stopwatch icon]: /docs/testing/ui-performance/images/observatory-icon.png
+[issues or feature requests]: {{site.github}}/dart-lang/sdk/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-observatory
+[`saveLayer`]: {{site.api}}/flutter/dart-ui/Canvas/saveLayer.html
+[timeDilation]: {{site.api}}/flutter/scheduler/timeDilation.html
+[Show performance data]: /docs/development/tools/android-studio#show-performance-data
+[Android Studio / IntelliJ]: /docs/development/tools/android-studio
+[rendering library]: {{site.api}}/flutter/rendering/rendering-library.html
+[`debugDumpRenderTree()`]: {{site.api}}/flutter/rendering/debugDumpRenderTree.html
```

Comment:
remove this link and tell people to view the render tree in the inspector instead.